specification/iotsecurity/resource-manager/Microsoft.IoTSecurity/preview/2021-07-01-preview/vulnerabilities.json

```diff
@@ -0,0 +1,297 @@
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "DeviceVulnerabilityListModel": {
+      "description": "List of IoT Device Vulnerabilities",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "List data",
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DeviceVulnerabilityModel"
+          }
+        },
+        "nextLink": {
+          "readOnly": true,
+          "type": "string",
+          "description": "When available, follow the URI to get the next page of data"
+        }
+      }
+    },
+    "DeviceVulnerabilityModel": {
+      "type": "object",
+      "description": "IoT Device Vulnerability",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "Device Vulnerability properties",
+          "$ref": "#/definitions/DeviceVulnerabilityPropertiesModel"
+        },
+        "systemData": {
+          "readOnly": true,
+          "type": "object",
+          "description": "Azure Resource Manager metadata containing createdBy and modifiedBy information.",
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/systemData"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ProxyResource"
+        }
+      ]
+    },
+    "DeviceVulnerabilityPropertiesModel": {
+      "type": "object",
+      "description": "IoT Device Vulnerability properties",
+      "properties": {
+        "id": {
+          "description": "The vulnerability ID",
+          "readOnly": true,
+          "type": "string",
+          "example": "CVE-2017-7797"
+        },
+        "title": {
+          "description": "The vulnerability title",
+          "readOnly": true,
+          "type": "string",
+          "example": "CVE-2017-7797"
+        },
+        "description": {
+          "description": "The vulnerability description",
+          "readOnly": true,
+          "type": "string",
+          "example": "This vulnerability affects the following vendors: Mozilla, Ubuntu. To view more details about this vulnerability please visit the vendor website."
+        },
+        "publishedDate": {
+          "description": "The date time of the vulnerability when he first published",
+          "readOnly": true,
+          "type": "string",
+          "format": "date-time",
+          "example": "2017-08-08T00:00:00Z"
+        },
+        "lastModifiedDate": {
+          "description": "The date time of the vulnerability when he changed lately",
+          "readOnly": true,
+          "type": "string",
+          "format": "date-time",
+          "example": "2018-98-18T00:00:00Z"
+        },
+        "severity": {
```

Comment:
Some properties are marked as `readOnly` while others don't ... is it on purpose? (`severity`, `score`)

Reply:
no. fixed.